management/authenticators.go

```diff
@@ -0,0 +1,329 @@
+package ziti
```

Comment:
these functions will be some of the first ones people explore, i'd really appreciate godoc on them once we agree on the api

Reply:
Sure sure.